tests/foreman/api/test_contentmanagement.py

```diff
@@ -995,3 +999,110 @@ def test_positive_sync_kickstart_repo(
         caps_pkgs = get_repo_files_by_url(caps_pkg_url)
         assert len(caps_pkgs)
         assert sat_pkgs == caps_pkgs
+
+    @pytest.mark.tier4
+    @pytest.mark.skip_if_not_set('capsule', 'clients')
+    def test_positive_sync_container_repo_end_to_end(
+        self, default_sat, capsule_configured, container_contenthost, module_org, module_product
+    ):
+        """Sync container repositories with schema1, schema2,
+        and both of them to the capsule and pull them to a
+        content host.
+
+        :id: 8fab295c-a9bf-4c00-b800-f721ae8f29fe
+
+        :steps:
+            1. Sync container repositories to Satellite.
+            2. Publish them in a CV, promote to Capsule's LCE.
+            3. Check the Capsule is synced without errors.
+            4. Pull the image from the Capsule to a host.
+
+        :expectedresults:
+            1. The container repo is successfully synced to the Capsule.
+            2. The image is successfully pulled from Capsule to a host.
+
+        :CaseLevel: Integration
+        """
+        upstream_names = [
+            'quay/busybox',  # v1
+            'foreman/busybox-test',  # v2
+            'foreman/foreman',  # v1+v2
```

Comment:
what do the v1 and v2 mean in these comments?

Reply:
It refers to the schema versions available in the image.